web/static/embed.go

```diff
@@ -42,3 +42,6 @@ var SlackWebContent embed.FS
 
 //go:embed twilio/connect
 var TwilioWebContent embed.FS
+
+//go:embed aws/connect
+var AWSWebContent embed.FS
```

Comment:
Please move this blocks above ChatGPT, not at the bottom of the file

Reply:
done